Feature/267 Restructuring of the wiki

kbwsoorensen opened the pull request:
Clone the repository and change to the 'feature/267' branch.
Then, run the wiki according to the guide described in:
https://github.com/aau-giraf/wiki/blob/feature/267/docs/Development/Wiki/running_wiki.md

Feedback to the restructuring and changes are most welcome.

Closes #267 
Closes #211 
Closes #249

Mailaur committed:
changes to structure

Mailaur committed:
Link correction

Mailaur committed:
Files moved

Mailaur committed:
Changed filenames

Mailaur committed:
Names changed

Mailaur committed:
Naming changed

Mailaur committed:
names changed

Mailaur committed:
Renaming

Mailaur committed:
Renaming

Mailaur committed:
Renaming files

Determinatoro committed:
More structure done

Determinatoro committed:
Draft for the Handover and Release Guide

Determinatoro committed:
Added Sprint Planning and Sprint Review description in Handover

Commit:
Restructuring of Review_Checklists folder

Split the folder into 2020F and 2020E.
Added the old checklists to 2020F.
Added the current checklists to 2020E.
Added a index.md to the Review_Checklists folder.
Fixed references to old checklists.

Commit:
Further restructuring of Review_Checklists and upgrade of mkdocs-awesome-pages-plugin to 2.4.0

Index.md edited.
.pages changes from 'arrange' to 'nav'.
This change is enabled by mkdocs-awesome-pages-plugin 2.4.0, so the requirements.txt file has been updated as such.

Commit:
Fixed rendered checklists.

Fixed the checklists so they are rendered properly on the wiki.

Determinatoro committed:
Handover/Sprint Events draft finished

Determinatoro committed:
.pages added in Handover/Tools and offline mode overview image added to the wiki resources

Commit:
Fixes to checklists.

Checklists respect the linter now.
Missing folder added to tools folder.

Determinatoro committed:
Changes to Sprint Events and Role Structure in 2020E Handover

Riasalit committed:
Started on updating Offline_mode

* Removed most of the outdated information
* Wrote headers which should work as todos
* Wrote a bit

Determinatoro committed:
Changes to Development/Apps

Determinatoro committed:
Changes in .pages

Commit:
Work on Process_Group tools.

Added .pages.
Added and wrote pull_requests_overview.md.

Riasalit committed:
Fixed headers todos and meged other fix

wuuu ruby -.-

Riasalit committed:
Merge branch 'feature/267' of https://github.com/aau-giraf/wiki into feature/267

LivHolm committed:
Discriptions of google calender and slack has been added

Commit:
Fixes for linter.

FIxed headers in google_calendar.md and slack.md.

Riasalit committed:
Mroe offline_database changes

Riasalit committed:
Merge branch 'feature/267' of https://github.com/aau-giraf/wiki into feature/267

Commit:
Update pull_requests_overview.md

pull_requests_overview.md finished.

Commit:
Further work on tools.

individual_pull_request.md has been added and written.
pull_request_overview.md had a minor correction.
Links in Review_Checklists\2020E\index.md has been fixed.

Commit:
Update .pages

Riasalit committed:
Wrote more offline

Commit:
Continued work on tools

Added issues_overview.md, and wrote it.

Determinatoro committed:
Changes to Development

Determinatoro committed:
Changes to Development/Web API

Commit:
Added sprint_points.md

Commit:
Merge branch 'feature/267' of https://github.com/aau-giraf/wiki into feature/267

Commit:
Linter fixes

Determinatoro committed:
Updates to Process Group Tools in Handover

Determinatoro committed:
Merge branch 'feature/267' of https://github.com/aau-giraf/wiki into feature/267

Commit:
Corrections

Commit:
Merge branch 'feature/267' of https://github.com/aau-giraf/wiki into feature/267

LivHolm committed:
Small changes to the text

Changes made in slack and google calender

Mailaur committed:
Review of Google Calendar

Commit:
Added example images and sprint_points.md

LivHolm committed:
Sheet used for sprint planning added

LivHolm committed:
Merge branch 'feature/267' of https://github.com/aau-giraf/wiki into feature/267

Mailaur committed:
Merge branch 'feature/267' of https://github.com/aau-giraf/wiki into feature/267

Mailaur committed:
Merge branch 'feature/267' of https://github.com/aau-giraf/wiki into feature/267

Mailaur committed:
Review of slack.md

Mailaur committed:
Update sprint_planning.md

Commit:
Update sheet_used_for_sprint_planning.md

Commit:
Merge branch 'feature/267' of https://github.com/aau-giraf/wiki into feature/267

Commit:
Refactoring of tools

Renamed a bunch of the tools.
Wrote the index.md for Tools root and Process_Group.

Commit:
Update index.md

Added navigation to Review_Checklist\index.md.

Commit:
Update google_calendar.md

Minor spelling fix.

Commit:
Update google_calendar.md

Calender -> Calendar

LivHolm committed:
updated overview in handover

added that advice can be found in the reports

Commit:
Added index to Review_Checklists

Commit:
Added sprint_retrospective_tool.md

Commit:
Update index.md

Fixed link to Getting_Started

Commit:
Update ownership_transfer.md

LivHolm committed:
Added release preparation event guide

Commit:
Minor corrections

Riasalit committed:
Final update on the Offline database page from giraf 9

Commit:
Update writing_wiki.md

Commit:
Update writing_wiki.md

Updated the guide for running the wiki locally, and added a guide for running markdownlint locally.

Commit:
Update writing_wiki.md

Fixed linter issues.

Determinatoro committed:
changes draft

Determinatoro committed:
Merge branch 'feature/267' of https://github.com/aau-giraf/wiki into feature/267

Determinatoro committed:
Getting Started and Development ready for review

Commit:
Update writing_wiki.md

Minor update before pull.

Commit:
Merge branch 'feature/267' of https://github.com/aau-giraf/wiki into feature/267

Determinatoro committed:
Changes in Development/Server

Determinatoro committed:
More changes to Server

Determinatoro committed:
Merge branch 'feature/267' of https://github.com/aau-giraf/wiki into feature/267

Commit:
Wiki done.

Added index.md.
Added pages.md.
Split writing_wiki.md into itself and running_wiki.md.
Updated Develoment\index.md.

Determinatoro committed:
Minor changes

Commit:
Added/fixed indexes and .pages files.

Commit:
Update README.md and wiki.

Fixed headers in wiki.

Commit:
Minor fix

Commit:
Update index.md

Refactor.

Commit:
Minor fixes

Determinatoro committed:
Update index.md in Handover

Determinatoro committed:
Added files from develop, updates headers in some files

Commit:
Minor fixes to endpoints

ghost commented:
Ignore the merge conflicts for now

Kaliahh commented:
Det er en KLAR forbedring over hvad der var før. Når vi lige sammenligner med det der er der nu, så har det her faktisk struktur. Vi har dog lige nogle kommentarer og forslag til ændringer:

#### Generelle ting:
- Måske bør api-clienten have et punkt under Development?
- About virker lidt random. Måske About skulle handle om hvad app'en er og hvordan den bliver udviklet (Fra et meget overorndet synspunkt)?
- Overviews bør nok beskrive lidt hvad det er man kan finde under de forskellige punkter. Lige nu er de bare lidt til overs, vi har jo oversigten til venstre der siger præcis det samme. Der behøver heller ikke stå at "This section gives an overview", for det er jo lidt det den gør.
- Måske widget testing ikke skulle stå under unit testing? Måske bare have én test fil, i stedet for at have en mappe med én fil i?
- Er det der står under Development faktisk rigtigt og up-to-date?


### Forslag til struktur:

#### Home

#### Getting Started
 - Links til de forskellige setup guides

#### Development
 - Hvordan skal man udvikle videre på app'en? Guidelines til udvikling.

#### UI Design
 - Prototyper
   - Beskrivelser af prototyper der ikke er blevet implementeret
   - Guide til Xd
 - Design guide
 - Licenses

#### Handover
 - Mere om vores proces. Hvordan vi har lavet Scrum events, hvilke roller vi har haft, gode råd, værktøjer vi har brugt

#### Releases

#### Security
 - GDPR
 - Risk assessment
 - Privacy policy

#### About
 - Hvem er vi, og hvad er motivationen for det vi gør? Formålet med app'en

Commit:
Added Security folder

Added Security folder to docs/
Moved contents from About to Security
Restructured Security contents with .pages
Added index.md files to Security where necessary

Commit:
Update index.md

Added About in About folder (stolen from https://giraf.cs.aau.dk/en/about-giraf/)

Commit:
Added UI Design folder

Added UI Design folder with .pages and index.md

Commit:
Renamed 'Security' folder to 'Legal'

Commit:
Update docs\index.md

Commit:
Moved Design_Guide from Development\Apps to UI_Design

Commit:
Major changes

Moved Prototypes to UI_Design and updated .pages and index.md
Changed test folder to just a test.md file, and updated .pages accordingly

Commit:
Updated index.md a lot of places

Added small descriptions of most of the sections when they are in an index.md file

Determinatoro committed:
Update sprint_retrospective.md

Commit:
Added quickstart links to development under Getting Started

ghost commented:
@Kaliahh vi har prøvet at re-restrukturere wikien igen efter din kommentar, hvis du/i vil kigge den igennem igen

Mustafa-ALD committed:
Create Issue_526.md

JeppeKLau committed:
Create issue418.md

Commit:
Merge conflict fixes (Incomplete)

Updated licences.md, icons.md, and typography.md to reflect develop branch.
(Merging develop into the branch proved to be a disaster, so manually updating the files instead)

Commit:
Remove .idea folder to prevent conflicts

Commit:
Update .gitignore

Commit:
Merge branch 'develop' into feature/267

Commit:
Merge conflict fixes

Commit:
Delete WeekplannerErrorHandlingGuidelines.md

Phantom file? It aint actually on GitHub? What will happen?

Mustafa-ALD committed:
Update Issue_526.md

Commit:
Linter fixes

Upped 'offline_mode.md' in .pages
Made convention and linter fixes to Issues 417 and 526.
Renamed the two issues to snake_case.

Commit:
Update issue_526.md

Minor linter fixes

Commit:
Merge branch 'feature/267' of https://github.com/aau-giraf/wiki into feature/267
